wg-automl/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# AutoML Working Group
+
+WG AutoML is responsible for all aspects of Automated Machine Learning technologies on Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the AutoML Working Group.
+
+## Meetings
+* Regular Katib Meeting (Asia & Europe friendly): [Wednesdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00 am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
```

Comment:
```suggestion
* Regular Katib Meeting (Asia & Europe friendly): [Wednesdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
```

Reply:
The `wgs.yaml` had a redundant space that was kept as is. Let me regenerate with the change in `wgs.yaml`